endpoints/openrtb2/auction.go

```diff
@@ -421,13 +420,26 @@ func validateUser(user *openrtb.User, aliases map[string]string) error {
 			}
 		} else {
 			// Return error.
-			return errors.New("request.user.ext object is not valid.")
+			return fmt.Errorf("request.user.ext object is not valid: %v", err)
 		}
 	}
 
 	return nil
 }
 
+func validateRegs(regs *openrtb.Regs) error {
+	if regs != nil && len(regs.Ext) > 0 {
+		var regsExt openrtb_ext.ExtRegs
+		if err := json.Unmarshal(regs.Ext, &regsExt); err != nil {
+			return fmt.Errorf("request.regs.ext is invalid: %v", err)
+		}
+		if regsExt.GDPR != nil && (*regsExt.GDPR < 0 || *regsExt.GDPR > 1) {
+			return errors.New("request.regs.ext.gdpr must be either 0 or 1.")
```

Comment:
Should this be `errors.New`?  I saw you change other error messages to `fmt.Errorf`.

Reply:
Yeah. `fmt.Errorf` lets you use formatting strings like `request.imp[%d], 3`. `errors.New` just accepts raw strings.

No formatting strings in this one, so.... `errors.New`